airflow/README.md

```diff
@@ -248,6 +255,42 @@ _Available for Agent versions >6.0_
 
 3. [Restart the Agent][7].
 
+<!-- xxz tab xxx -->
+<!-- xxx tab "Containerized" xxx -->
+
+#### Containerized
+
+For containerized environments, see the [Autodiscovery Integration Templates][13] for guidance on applying the parameters below.
+
+##### Metric collection
+
+| Parameter            | Value                 |
+|----------------------|-----------------------|
+| `<INTEGRATION_NAME>` | `airflow`             |
+| `<INIT_CONFIG>`      | blank or `{}`         |
+| `<INSTANCE_CONFIG>`  | `{"url": "%%host%%"}` |
+
+##### Log collection
+
+_Available for Agent versions >6.0_
+
+Collecting logs is disabled by default in the Datadog Agent. To enable it, see [Kubernetes log collection documentation][14].
+
+| Parameter      | Value                                                 |
+|----------------|-------------------------------------------------------|
+| `<LOG_CONFIG>` | `{"source": "airflow", "service": "<YOUR_APP_NAME>"}` |
+
+##### Kubernetes
+
+Tips for Kubernetes installations:
+
+- Customize the Airflow configuration with [pod annotations][14].
+- When modifying `airflow.cfg`, `statsd_host` should be set to the IP address of the Kubernetes node.
+- Use the environment variable `DD_DOGSTATSD_MAPPER_PROFILE` in your helm or daemonset to set `dogstatsd_mapper_profiles` in your `datadog.yaml`.
```

Comment:
Note that using environment variable `DD_DOGSTATSD_MAPPER_PROFILE` is not available yet, but there is an open PR: https://github.com/DataDog/datadog-agent/pull/6560

Reply:
@ruthnaebeck The PR is merged and the feature will be available with 7.24.0

We might want to add back this line when 7.24.0 is released (possibly in another PR).